Library/Homebrew/formula_installer.rb

```diff
@@ -55,6 +56,13 @@ def initialize(formula)
     @pour_failed   = false
   end
 
+  # called by install/reinstall/upgrade when no build tools are available
+  def self.check_build_flags
```

Comment:
@dunn: Any followup thoughts on this? I think `prevent_build_flags` is a good alternative, but I defer to you on this :smile: